mysensors/__init__.py

```diff
@@ -225,7 +225,7 @@ def _load_pickle(self, filename):
     def _save_json(self, filename):
         """Save sensors to json file."""
         with open(filename, 'w') as file_handle:
-            json.dump(self.sensors, file_handle, cls=MySensorsJSONEncoder)
+            json.dump(self.sensors, file_handle, cls=MySensorsJSONEncoder, indent=4)
```

Comment:
You need to break the line, to conform to PEP rules, ie <80 characters per line.
```py
json.dump(
    self.sensors, file_handle, cls=MySensorsJSONEncoder, indent=4)
```